index.js

```diff
@@ -15,7 +15,11 @@ module.exports = function(precompile) {
             throw file.errorWithNode(node, msg);
           }
 
-          this.remove();
+          if (typeof this.dangerouslyRemove === 'function') {
+              this.dangerouslyRemove();
+          } else {
+              this.remove();
```

Comment:
This looks good! One tiny nitpick: can you add a comment stating why we are doing this and maybe that this can be removed once we only support `babel => v5.5.0`... ?